docs/features/ai/ai-errors.md

```diff
@@ -0,0 +1,22 @@
+---
+id: ai-errors
+title: "Yoast AI error messages"
+sidebar_label: AI errors
+description: This documentation provides information about the errors that the Yoast AI feature shows.
+---
+
+This documentation provides information about the errors displayed by the [Yoast AI](https://yoast.com/features/ai-generated-titles-and-meta-descriptions/#utm_medium=software&utm_source=yoast-seo&utm_term=ai-feature-page&utm_content=ai-modal) features of the [Yoast SEO](https://yoast.com/wordpress/plugins/seo/) plugin.
+
+## SEO analysis required
+Our Yoast AI optimization tool, requires the SEO analyses to be enabled in the settings of the plugin. When a premium user try to make use of the AI features whithout having the SEO analyses enabled, the following error message will be shown: 
```

Comment:
Some spelling errors:
- the comma in the first sentence is unnecessary.
- `analyses` is plural, please use the singular form.
- `whitout` => `without`
- `premium user` => `Yoast SEO Premium user` or just `user`
- `try` => `tries`